### PR TITLE
Better cleanup after device remove

### LINKED ISSUE #400: State stays after removing device
Happened after removing last device from the account. Not sure if it's relevant.

### PR DESCRIPTION
**Pull request by Abyss777**
fix #400 

Also noticed that accuracy and live route stayed after device removed

### CODE REVIEW

File: web/app/view/StateController.js [hunk 1]
```diff
@@ -49,6 +49,9 @@ Ext.define('Traccar.view.StateController', {
                 '#AttributeAliases': {
                     add: 'updateAliases',
                     update: 'updateAliases'
+                },
+                '#Devices': {
+                    remove: 'removeDevice'
```
Comment (tananaev) [inline web/app/view/StateController.js]: I'm just wondering, why it doesn't get de-selected? Do we really need to explicitly do this?

Comment (Abyss777) [inline web/app/view/StateController.js]: Hm, you are right, via de-select it will look more consequent.

**Comment by Abyss777** [comment]
Replaced commit.

Devices grid correctly handles device removing and fires `onSelectionChange` event.
Just fire our `deselectfeature` event if nothing selected.

Also handle `deselectfeature` event only from `map`, to avoid double call `deselectFeature` function.

**Comment by tananaev** [comment]
Looks good, thanks.

**Pull request merged**